README.md

```diff
@@ -1289,6 +1289,27 @@ Other Style Guides
 
   - [15.4](#15.4) <a name='15.4'></a> For more information see [Truth Equality and JavaScript](http://javascriptweblog.wordpress.com/2011/02/07/truth-equality-and-javascript/#more-2108) by Angus Croll.
 
+  - [15.5](#15.5) <a name='15.5'></a> Ternaries should not be nested and generally be single line expressions.
```

Comment:
The only case that's been brought up is when you want to use `const` but have a multi-part condition that you don't want to evaluate unless you have to - it gets really ugly using a `let` and an `if` block, or breaking up the condition.

That said, I still prefer only ever using single-line ternaries.